search_that_hash/cracker/fast_mode_mod/runner.py

```diff
@@ -52,6 +54,13 @@ def fast_crack(self):
                 self.results.append({chash: result})
                 continue
 
+            for type in types:
+                types_to_push.append(type["name"])
+                if len(types_to_push) == 5:

```

Comment:
why 5? whats this?

Reply:
5 so that it doesn't push to the DB with 100 possible types

Reply:
Perhaps you should add a comment explaining that.

Reply:
👍🏻